### PR TITLE
Add csrf and update submit post

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
1. Extracting csrf token
1. Add csrf token to request body
1. Update submit to post to next block

### How to review
1. Run survey runner in debug
1. Run performance test against $host with one user
```
docker run \                                   
    -e HOST=$host \
    -e PROTOCOL=http \
    -e PORT=5000 \
    -e USERS=1 \
    -e REPEAT=1 \
    -v ~/testResults:/eq-performance-test/results \
   eq-performance-test
```
1. Check all POSTs but one is valid and answers are submitted

### CODE REVIEW

File: survey_runner_jmeter.rb [hunk 1]
```diff
@@ -123,19 +129,23 @@ def post_page_1_filled()
...
 
 def post_final_submission()
 
-    submit name: 'POST final submission', url: '${url}submit-answers',
-            fill_in: { "action[submit_answers]": "" } do
+    submit name: 'POST final submission', url: '${url}',
+            fill_in: { "action[submit_answers]": "",
+            "csrf_token" => "${csrf_token}" } do
         assert contains: ['Submission Successful', 'Transaction ID'], scope: 'main'
+        extract_csrf
```
Comment (collisdigital) [inline survey_runner_jmeter.rb]: Any need to extract it; survey should be finished now?

Comment (benjefferies) [reply survey_runner_jmeter.rb]: Nope just got copy-paste happy

**Review by collisdigital — APPROVED** [review]
Ran locally 500 users; all succeeded. Might be worth adding to README.md that you should use lo0 via ifconfig on a Mac?

**Comment by benjefferies** [comment]
Fair point @collisdigital. Done.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
When I run it I get a failure:
`1489402400026|26|POST page 1 (filled)|200|OK|ThreadGroup 1-1|false|Test failed: text expected to contain /Your responses/|10188|1|1|26|1|1|null`

**Review by ajmaddaford — APPROVED** [review]
I hadn't run docker build! Works for me now 😃

**Pull request merged**